tests/dummy/app/pods/button/template.hbs

```diff
@@ -0,0 +1,387 @@
+<div class="dummy-notification-container">
+  {{#each notifications as |notification|}}
+      {{notification-message notification=notification}}
+  {{/each}}
+</div>
+
+<h2>frost-button</h2>
+{{#frost-checkbox
+  checked=vertical
+  on-input=(action "toggleVertical")
+}}
+  Vertical
+{{/frost-checkbox}}
+
+<br/>
+
+<div class="section">
+  <div class="example">
+    <div class="title">Primary</div>
+    <div class="code">
```

Comment:
will need a change to demo on top then code snippet below to be consistent with others